src/main/java/com/example/may/cloudfoundry/xsuaa/model/XsuaaClientCredentials.java

```diff
@@ -0,0 +1,22 @@
+package com.example.may.cloudfoundry.xsuaa.model;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Evelina Tubalets
+ */
+@Data
+@Component
+@ConfigurationProperties(prefix = "destination-service")
```

Comment:
these xsuaa credentials must be generic, not tightly coupled to destination service